sdk/keyvault/azure-security-keyvault-secrets/src/main/java/module-info.java

```diff
@@ -10,4 +10,5 @@
     opens com.azure.security.keyvault.secrets to com.azure.core, com.fasterxml.jackson.databind;
     opens com.azure.security.keyvault.secrets.implementation to com.azure.core, com.fasterxml.jackson.databind;
     opens com.azure.security.keyvault.secrets.models to com.azure.core, com.fasterxml.jackson.databind;
+    exports com.azure.security.keyvault.secrets.implementation;
```

Comment:
why are we exporting implementation package?

Reply:
intellij auto adds that.
Is part of PR cleanup TODOs.